resources/META-INF/plugin.xml

```diff
@@ -299,6 +299,18 @@
   <!-- See: http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html -->
 
   <actions>
+    <group id="Flutter.InspectorActions">
+      <action id="Flutter.JumpToTypeSource" class="io.flutter.inspector.JumpToTypeSourceAction"
+              description="Jump to Type Source"
+              text="Jump to Type Source">
+        <keyboard-shortcut keymap="$default" first-keystroke="shift F4"/>
+      </action>
+      <action id="Flutter.JumpToSource" class="io.flutter.inspector.JumpToSourceAction"
+              text="Jump to Source">
+        <keyboard-shortcut keymap="$default" first-keystroke="control DOWN"/>
```

Comment:
If this is not frequently used, we may not want a keybinding.

Reply:
If this doesn't conflict (as per the item above), then no real need to remove.

Reply:
I think we might as well be consistent with the existing debugger panel keyboard shortcuts that do the same thing. Personally I find having keybindings for these handy as jumping to the source is really handy.

Reply:
makes sense